examples/device/net_lwip_webserver/src/lwipopts.h

```diff
@@ -49,7 +49,9 @@
 
 #define TCP_MSS                         (1500 /*mtu*/ - 20 /*iphdr*/ - 20 /*tcphhr*/)
 #define TCP_SND_BUF                     (2 * TCP_MSS)
+#ifndef TCP_WND
```

Comment:
Note I should probably comment on this... there are pre-existing overrides of this to `2*TCP_MSS` in the CMakeLists.txt for the net_lwip_webserver example. This was causing a compiler warning, as this line just trashes it. So either my change is correct, or the override is unnecessary!

Reply:
The defined in cmakelist.txt is added in this commit
https://github.com/hathach/tinyusb/commit/95f2478146ef83d18b52408c0692222bf2a15053#diff-12cfd80973cd9c09e8493c08a1eb623365231a9e2b0db14d109e9b6836985830R71-R74

```
PBUF_POOL_SIZE=2
HTTPD_USE_CUSTOM_FSDATA=0
```

are also redefined with the same values, maybe we should drop it altogether in cmakelist and adjust/use value in the header file only for consistency